dags/maximo_geo_emergency_mgmt_email_parser.py

```diff
@@ -0,0 +1,69 @@
+import os
+from pendulum import datetime, duration
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Parse the most recent email received containing Maximo Emergency Management data",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "execution_timeout": duration(minutes=20),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+REQUIRED_SECRETS = {
+    "AWS_DEFAULT_REGION": {
+        "opitem": "Maximo Geo Integrations",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Default Region",
+    },
+    "AWS_ACCESS_KEY_ID": {
+        "opitem": "Maximo Geo Integrations",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Access Key",
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+        "opitem": "Maximo Geo Integrations",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Secret Access Key",
+    },
+}
+
+# fmt: off
+with DAG(
+    dag_id=f"maximo_geo_emergency_mgmt_email_parser_{DEPLOYMENT_ENVIRONMENT}",
+    default_args=default_args,
+    schedule_interval="*/30 * * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
+    tags=["repo:dts-maximo-geo-integration", "maximo", "geo", "emergency-management", "email-parser"],
+    catchup=False,
+) as dag:
+# fmt: on
+
+    @task(
+        task_id="get_env_vars",
+        execution_timeout=duration(seconds=30),
+    )
+    def get_env_vars():
+        from utils.onepassword import load_dict
+
+        return load_dict(REQUIRED_SECRETS)
+
+    env_vars = get_env_vars()
+
+    DockerOperator(
```

Comment:
We haven't been super consistent across DAGs but this could use the `docker_conn_id="docker_default"` param.

Reply:
+1, I've been adding them when I update a dag, but we still have a good amount that are missing them. There's an issue floating around for it....